src/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ruby:2.5.1
+
+RUN mkdir -p /usr/src/ruby_first
+WORKDIR /usr/src/ruby_first
```

Comment:
From docker file:
```
The WORKDIR instruction can resolve environment variables previously set using ENV. You can only use environment variables explicitly set in the Dockerfile. For example:
```
So instruction on line 6 will not be available.
you probably need to move this line below

Reply:
fixed